src/test/resources/gradle_plugin/pipeline_test_build_scan.txt

```diff
@@ -0,0 +1,12 @@
+stage ('build_Project') {
+    steps {
+        script {
+           if(isUnix()) {
```

Comment:
As of Gradle plugin `2.17.1` setting the path shouldn't be necessary if the tool is already selected. I think this PR should have fixed it: https://github.com/jenkinsci/gradle-plugin/pull/42